src/components/molecules/Cart.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import CartItem from '../atoms/CartItem';
+import NavButton from '../atoms/NavButton';
+
+function Cart({ count, value, icon }) {
```

Comment:
would be better to follow ES6 syntax while creating functional components. for ex. const Cart = (props) => {}

Reply:
Yes, I would've liked that too, but for some reason ESLINT with Airbnb ruleset doesn't like anything else, except functions. (although it is ok with const = (....) => {....} in pure JS)

Reply:
https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless